src/CMakeLists.txt

```diff
@@ -252,7 +252,7 @@ endif()
 # ==============================================================================
 option(BOOST_NO_CXX11 "if Boost is compiled without C++11 support (as it is often the case in OS packages) this must be enabled to avoid symbol conflicts (SCOPED_ENUM)." OFF)
 set(ALICEVISION_BOOST_COMPONENTS atomic container date_time filesystem graph log log_setup program_options regex serialization system thread timer)
-find_package(Boost 1.60.0 QUIET CONFIG COMPONENTS ${ALICEVISION_BOOST_COMPONENTS})
+find_package(Boost 1.60.0 QUIET COMPONENTS ${ALICEVISION_BOOST_COMPONENTS} unit_test_framework)
```

Comment:
```cmake
if(ALICEVISION_BUILD_TESTS)
    list(APPEND ALICEVISION_BOOST_COMPONENTS unit_test_framework)
endif()
```

Reply:
But unit_test_framework should not be exported in the Config as it's not needed when AliceVision is used as an external library.

Reply:
true.
But if you use it as 3rd party normally you don't build the tests that are set to OFF by default.
Nevertheless, we need some conditions here, because for the same reason you don't need to find the test framework if you are not building the tests

Reply:
sure